sdk/spring/spring-cloud-azure-test-eventhubs-binder/src/test/java/com/azure/spring/sample/eventhubs/binder/EventHubsBinderSyncModeIT.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.spring.cloud.stream.binder.eventhubs;
+package com.azure.spring.sample.eventhubs.binder;
```

Comment:
nit: I think we should not use the package sample here.

Reply:
I will change it in the next PR.